src/libs/thorvg/tvgSwRenderer.cpp

```diff
@@ -446,18 +447,14 @@ bool SwRenderer::target(pixel_t* data, uint32_t stride, uint32_t w, uint32_t h,
 
 bool SwRenderer::preRender()
 {
-#if LV_USE_DRAW_VG_LITE && LV_USE_VG_LITE_THORVG
```

Comment:
Please retain the modifications; otherwise, the display of vg-lite emulator will be affected